ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -111,7 +111,7 @@ open class ContactCollectionView: UICollectionView {
     }
 
     @objc private func updateHeightConstraint() {
-        let height = UIApplication.shared.preferredContentSizeCategory.contactHeight(size: size.contactViewSize) + 2 * Constants.verticalInset
+        let height = UIScreen.main.traitCollection.preferredContentSizeCategory.contactHeight(size: size.contactViewSize) + 2 * Constants.verticalInset
```

Comment:
Can you explain in the PR description why this change is needed?

Reply:
Either way we might not want to use UIScreen... #77

Reply:
Thanks for your review. I've updated.